cmd/xl-v1-healing.go

```diff
@@ -138,7 +138,7 @@ func xlShouldHeal(partsMetadata []xlMetaV1, errs []error) bool {
 			return true
 		}
 		if errs[index] != nil {
-			continue
+			return true
```

Comment:
Was it decided to ``return true`` only for err == errDiskNotFound and mismatching modtime?